distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestSpatial.java

```diff
@@ -345,6 +354,27 @@ public void testCswFuzzyTextQuery()
         performQueryAndValidateExpectedResults("CswFuzzyTextQuery");
     }
 
+    @Test
+    public void testGeoCoderPlugin() throws Exception {
+        String uuid = UUID.randomUUID()
+                .toString()
+                .replaceAll("-", "");
```

Comment:
Is there a reason we need to generate an id?  Doesn't ingestCswRecord return the id, if we need it?  And it seems like we don't need it.

Just concerned if id format were changed, and a validator rejected this ingest, it would break the test.